docs/getting-started/install-scylla/oss-install-azure.rst

```diff
@@ -0,0 +1,99 @@
+.. |VERSION| replace:: 5.2
+.. |IMAGE-DEFINITION| replace:: scylla-5.2
+.. |IMAGE-VERSION| replace:: 5.2.1
+.. |GALLERY-NAME| replace:: 6c268694-47ab-43ab-b306-3c5514bc4112
+
+==========================
+Launch ScyllaDB on Azure
+==========================
+
+This article will guide you through self-managed ScyllaDB deployment on Azure. For a fully-managed deployment of ScyllaDB 
+as-a-service, see `ScyllaDB Cloud documentation <https://cloud.docs.scylladb.com/>`_.
+
+With the right setting, ScyllaDB image can be used for production. For the ScyllaDB Image documentation, see 
```

Comment:
> With the right setting, 

Why do we even need this disclaimer? It's vague and doesn't help much.

Reply:
It could be because of the non-production snitch. Wait for info.

Reply:
Removing it for now. If the info about the snitch is needed, it should be added as one of the final steps.